var/spack/repos/builtin/packages/py-py-spy/package.py

```diff
@@ -19,7 +19,7 @@ class PyPySpy(Package):
     # Need to figure out how to manage these with Spack once we have a
     # CargoPackage base class.
     depends_on('rust', type='build')
-    depends_on('unwind')
+    depends_on('libunwind components=ptrace')
```

Comment:
Can you try this:
```suggestion
    depends_on('unwind')
    depends_on('libunwind components=ptrace', when='^libunwind')
```
and check if it works? Your PR makes this package unbuildable on `darwin`.

Reply:
Tested on the suse that was failing the build and it loads the correct version of libunwind. Thanks for the suggestion.